src/WebApp.Web/Features/WeatherForecast/AddWeatherForecast.cs

```diff
@@ -8,6 +9,35 @@ public record AddWeatherForecast(DateOnly Date, int TemperatureC, int WindSpeed,
 {
 }
 
+[UsedImplicitly]
+public class AddWeatherForecastValidator : AbstractValidator<AddWeatherForecast>
```

Comment:
All these checks can be done using Data Annotations. We need to come up with a more realistic scenario of the domain validation logic, e.g. to call database and check the db state.

Reply:
Replaced simple validation to data annotation attributes. Added more complex validation logic to fluent validation